src/gui/properties/propertieswidget.cpp

```diff
@@ -76,7 +76,6 @@ PropertiesWidget::PropertiesWidget(QWidget *parent)
     , m_ui(new Ui::PropertiesWidget())
 {
     m_ui->setupUi(this);
-    setAutoFillBackground(true);
```

Comment:
Better do it in the .ui file instead.

Reply:
Unsure I understand what you're implying. The default value for setAutoFillBackground is false, so removing it will have it remain false. It's not set in the UI files.

Reply:
You're right. I somehow totally misunderstood the diff.